docs/self-managed/platform-deployment/helm-kubernetes/guides/ingress-setup.md

```diff
@@ -164,11 +166,16 @@ tasklist:
     className: nginx
     host: "tasklist.camunda.example.com"
 
-zeebe-gateway:
+zeebeGateway:
   ingress:
-    enabled: true
-    className: nginx
-    host: "zeebe.camunda.example.com"
+    rest:
+      enabled: true
+      className: nginx
+      host: "zeebe.camunda.example.com"
+    grpc:
+      enabled: true
+      className: nginx
+      host: "zeebe-grpc.camunda.example.com"
```

Comment:
:thought_balloon: Here we need two different sub-domains unfortunately :( Open to suggestions!

Reply:
I think when the kubernetes Gateway api becomes more widespread, I think it'll be easier to address this problem.